backend/routes/index.js

```diff
@@ -1693,6 +1623,42 @@ module.exports = function() {
     }
   });
 
+  router.get("/user/balance", (req, res) => {
+    try {
+      MongoClient.connect(dbUrl, async (err, db) => {
+        let dbo = db.db(dbFolder);
+        let user = await dbo.collection("users").findOne({"profile.accountName": req.user })
+        let data = {
+          balance: 0,
+          creditLimit: 0,
```

Comment:
I don't get why you define this up here if you only override it again later on 
            data.creditLimit = data.creditLine 
below?